### ISSUE TITLE
Allowing people to save preferences

### ISSUE THREAD

**Issue by bachrach44** [Sefaria/Sefaria-Project#94]
Feature suggestion: Since people have (or can have) profiles already, why not let them set some preferences? For example:
1 - Whether they want to see the website's interface in Hebrew or English. (Equivalent to the aleph/A button on the homepage).
2 - Whether they prefer the text to be in English, Hebrew, or bilingual. (Equivalent to the button on the pages of text).
3 - whether they like bilingual text to be English on the left of Hebrew on the left.
4 - Which font they prefer.
etc.

Basically anything that is right now an option that can be selected via dropdowns and buttons should also be configurable by an individual as their personal default option.
